staging/src/k8s.io/client-go/tools/reference/ref.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package ref
+package reference
```

Comment:
I would like to get rid of the pkg dir. Not much sense to have a single helper package there. Why not tools?